jina/clients/sugary_io.py

```diff
@@ -1,3 +1,4 @@
+"""A module for sugary API wrapper around the clients."""
```

Comment:
this may cause problems because this `_copywright_`is added automatically by `jina dev bot`

Reply:
https://jinaai.slack.com/archives/C01LX54KWER/p1612889874047100?thread_ts=1612886988.046200&cid=C01LX54KWER